packages/x-ui-collections-bundle/src/graphql/Collection.ts

```diff
@@ -360,6 +379,12 @@ export abstract class Collection<T = null> {
     return this.hybridFind(false, query, body);
   }
 
+  protected optimisticUpdates = true;
+
+  public enableOptimisticUpdates(option: boolean) {
```

Comment:
I don't know if this is a good approach, at what step would you actually call this enableOptimisticUpdates? In every component? Rather have something: isOptimisticUpdateEnabled() { return true }